go/login/server/example-server/cmd/utils.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os/exec"
+	"strings"
+
+	"../../../../glome"
+	"../../../server"
+
+	"gopkg.in/yaml.v2"
+)
+
+func decodeString(s string) []byte {
+	b, err := hex.DecodeString(s)
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
+	return b
+}
+
+// yamlkey is an auxiliary type for communication with go-yaml library.
+type yamlkey struct {
+	Key   string
+	Index uint8
+}
+
+// readKeys reads keys from provided package
+func readKeys(filename string) map[string]yamlkey {
```

Comment:
In general functions that use `log.Fatalf` are untestable. You should do the normal thing of returning an error using `fmt.Errorf` and handle it at the call site.